README.md

```diff
@@ -1,4 +1,19 @@
-meku
-====
+# meku
+Kansallinen audiovisuaalinen instituutti Kuvaohjelmien luokittelu- ja valvontajärjestelmä.
 
-Kuvaohjelmien luokittelu- ja valvontajärjestelmä
+## Setting up the development environment
+Clone the repository and run the following commands. Note: The project runs well with `Node v16`.
+```
+$ npm install
+```
+Create Mongo DB
+```
+$ docker run --name kavi-mongo -p 27017:27017 -d mongo:3.6.12
+```
+Populate demo users and start the project
+```
+$ cd scripts
+$ node add-demo-users.js
+$ npm start
```

Comment:
Pitääkö toi `npm start` ajaa `scripts` kansiossa vai pitäiskö tässä ensin olla `cd ..`?

Ehkä laittaisin myös erikseen `Populate demo users` ja `Start the project` kohdat, koska tuoa populate pitää varmaan ajaa vain ensimmäisen kerran kun tuota pystyttää tai jos poistaa docker volumen. `npm start` taas pitää ajaa aina kun käynnistää sovelluksen.

Reply:
Hyviä pointteja. Eriytän ne.